app/assets/stylesheets/darkswarm/cart-dropdown.css.scss

```diff
@@ -0,0 +1,115 @@
+@import "mixins";
+@import "variables";
+@import "branding";
+
+.expanding-sidebar.cart-sidebar {
+  .background {
+    z-index: 150;
+  }
+
+  .sidebar {
+    padding: $topbar-height 0 0;
+    background-color: $white;
+    z-index: 160;
+
+    @include breakpoint(desktop) {
+      padding: $mobile-nav-height 0 0;
+    }
+  }
+
+  .cart-header {
+    padding: 1em;
+
+    .title {
+      color: $grey-800;
+      margin: 0;
+    }
+
+    .close {
+      color: $grey-500;
+      float: right;
+
+      i {
+        vertical-align: middle;
+      }
+    }
+  }
+
+  .cart-content {
+    margin-bottom: $sidebar-footer-height + 2em;
+
+    .cart-empty {
+      text-align: center;
+      padding-top: 10em;
+      width: 100%;
+
+      p {
+        font-size: 1.5em;
+      }
+    }
+  }
+
+  .go-shopping {
+    display: none;
+    padding: 0 1.5em;
+
+    @include breakpoint(mobile) {
+      display: inline-block;
+    }
+  }
+
+  table {
+    width: 100%;
+    border: 0;
+    border-spacing: 0;
+    margin: 0;
+
+    tr.product-cart {
```

Comment:
Is there a reason to specify the `tr`. Avoiding it will have the same result but lower coupling, meaning that things will be a bit more changeable, which is always desirable. The same applies to `tr.bought td` below.